docs/install.rst

```diff
@@ -18,39 +18,71 @@ Requirements
 PlasmaPy requires Python version 3.7 or newer.
 PlasmaPy requires the following packages for installation:
 
-- `NumPy <http://www.numpy.org/>`_ 1.18.1 or newer
-- `SciPy <https://www.scipy.org/>`_ 1.2 or newer
-- `Astropy <http://www.astropy.org/>`_ 4.0 or newer
+- `NumPy <https://www.numpy.org/>`_ — 1.18.1 or newer
+- `SciPy <https://www.scipy.org/>`_ — 1.2 or newer
+- `Astropy <https://www.astropy.org/>`_ — 4.0 or newer
+- `pandas <https://pandas.pydata.org/>`_ — 1.0 or newer
+- `xarray <http://xarray.pydata.org>`_ — above 0.14 but less than 0.16
+- `tqdm <https://tqdm.github.io/>`_ — 4.56 or newer
+- `colorama <https://pypi.org/project/colorama/>`_ — 0.3 or newer
+- `cached_property <https://pypi.org/project/cached-property/>`_ — 1.5.2 or newer
```

Comment:
Why is `cached_property` not in our `setup.cfg`?

Reply:
https://github.com/PlasmaPy/PlasmaPy/blob/af07d43f0e1091388fa8a98e84428ddf96a3d655/setup.cfg#L35 shows up for me!